src/theme/table/stickyHeader/index.js

```diff
@@ -1,34 +1,12 @@
 import React from 'react';
-import LoadLatestButton from './loadLatestButton';
 import styles from './stickyHeader.css';
 
 const StickyHeader = ({
   title,
-  button,
-  error,
   filters,
-  scrollToSelector,
 }) => (
   <div className={styles.header}>
     <h1>{title}</h1>
-    {// istanbul ignore next
-      button && !error && (
```

Comment:
@ikem-legend it's optional by passing the button, so we can have it for now till we remove the usage from all place

Reply:
@soroushm It's only used in two places `blocks` and `transactions` and we've already implemented the changes in the `blocks` domain. Once we work on the `transactions` domain, we'll update it there too